File: 2MONTH_LESSON2-master/app/src/main/java/com/example/a2_month_lesson2/ThirdFragment.java
package com.example.a2_month_lesson2;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import org.w3c.dom.Text;

import java.util.SplittableRandom;

public class ThirdFragment extends Fragment {

    private TextView tvResult3;

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_third, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        initView();

        String result_et = getArguments().getString(SecondFragment. KEY_EDIT_TEXT);
        Integer result_age = getArguments().getInt(SecondFragment.KEY_AGE);

        //tvResult3.setText(result_age.toString());
        tvResult3.setText(result_age);
    }

    private void initView() {
        tvResult3 = requireActivity().findViewById(R.id.tv_result3);

    }
}

File: 2MONTH_LESSON7-master/app/src/main/java/com/example/a2_monthlesson7/ThirdFragment.kt
package com.example.a2_monthlesson7

import android.annotation.SuppressLint
import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.example.a2_monthlesson7.databinding.FragmentSecondBinding
import com.example.a2_monthlesson7.databinding.FragmentThirdBinding

class ThirdFragment : Fragment() {

    private lateinit var binding: FragmentThirdBinding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        binding = FragmentThirdBinding.inflate(layoutInflater)
        return binding.root
    }

    @SuppressLint("SetTextI18n")
    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        /*val username = arguments?.getString(SecondFragment.USERNAME)
        val password = arguments?.getString(SecondFragment.PASSWORD)
        binding.tvUsernamePassword.setText("$username $password")*/

        val user = arguments?.getSerializable(SecondFragment.USER) as User
        binding.tvUsernamePassword.setText("Username: ${user.username} Password: ${user.password}")
    }
}